FirstProject/feedback.html

```diff
@@ -0,0 +1,185 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="x-ua-compatible" content="ie=edge">
+        <title>First App</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <!--Here is established the connection between HTML file and CSS file-->
+        <link rel="stylesheet" href="css/main.css">
+
+        <!--Here is the path to our images directory-->
+        <link rel="stylesheet" href="img/">
+
+        <!-- Latest compiled and minified CSS -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+
+        <!-- jQuery library -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+        <!-- Latest compiled JavaScript -->
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
+        <!--Adding style for mobile devices/ tablets / large monitors-->
+        <style>
+            /*Screen between 320px and 520px*/
+            @media screen and (max-width: 520px) and (min-width: 320px) {
```

Comment:
Uuuu! Media queries! Nice! Could you explain how media queries work, in your own words?